exercises/practice/dominoes/tests/dominoes.rs

```diff
@@ -153,7 +153,7 @@ fn need_backtrack() {
 #[test]
 #[ignore]
 fn separate_loops() {
-    let input = &[(1, 2), (2, 3), (3, 1), (1, 1), (2, 2), (3, 3)];
```

Comment:
Instead of editing this test, please add a new test with your input.

Reply:
Done as you suggest